src/sentry/static/sentry/app/views/settings/account/accountNotifications.jsx

```diff
@@ -14,28 +14,43 @@ import {PanelFooter} from 'app/components/panels';
 import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
 import accountNotificationFields from 'app/data/forms/accountNotificationSettings';
 
-const FINE_TUNE_FOOTERS = {
-  Alerts: {
-    text: 'Fine tune alerts by project',
-    path: 'alerts/',
-  },
-  'Workflow Notifications': {
-    text: 'Fine tune workflow notifications by project',
-    path: 'workflow/',
-  },
-  'Email Routing': {
-    text: 'Fine tune email routing by project',
-    path: 'email/',
-  },
-  'Weekly Reports': {
-    text: 'Fine tune weekly reports by organization',
-    path: 'reports/',
-  },
-  'Deploy Notifications': {
-    text: 'Fine tune deploy notifications by organization',
-    path: 'deploy/',
-  },
-};
+const FINE_TUNE_FOOTERS = Object.fromEntries([
+  [
+    t('Alerts'),
```

Comment:
Should we be translating the keys as well?

Reply:
This is broken *because* we're not translating the keys, the keys are coming from: https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/data/forms/accountNotificationSettings.tsx#L64

Reply:
We need to because the titles we use to look these footers up comes from `formGroups` in `data/forms/accountNotificationSettings.tsx`, which are translated